File: NastyHacks.java
import java.util.*;

class NastyHacks {
  public static void main(String[] args) {
    Scanner kbd = new Scanner(System.in);
   int n;
   int r,e,c;
   n=kbd.nextInt(); 
   for(int i=0;i<n;i++){
     r=kbd.nextInt();
     e=kbd.nextInt();
     c=kbd.nextInt();
     if(e-c==r){
      System.out.println("does not matter");
     }
     else if(e-c>r){
       System.out.println("advertise");
     }
     else{
       System.out.println("do not advertise");
     }
   }
   
  
  }
}

File: nastyhacks.cpp
#include <iostream>
using namespace std;
int main(){
   int n;
   int r,e,c;
   cin>>n; 
   for(int i=0;i<n;i++){
     cin>>r>>e>>c;
     if(e-c==r){
       cout<<"does not matter"<<endl;
     }
     else if(e-c>r){
       cout<<"advertise"<<endl;
     }
     else{
       cout<<"do not advertise"<<endl;
     }
   }
   
  }
